.vscode/settings.json

```diff
@@ -26,11 +26,13 @@
   },
 
   "[javascript]": {
-    "editor.formatOnSave": true
+    "editor.formatOnSave": true,
+    "editor.defaultFormatter": "esbenp.prettier-vscode"
```

Comment:
Is it possible to set the prettier version in config file as well?

Reply:
This plugin uses the correct version of prettier (the one configured in package.json) if it's installed. So if `./bin/lint --check-deps-only` runs without an issue it should use that version.

Reply:
nice!